Postman/Postman.js

```diff
@@ -107,111 +101,108 @@ Seriesコレクションから新刊のBookListドキュメントを抽出する
 @param { FUnction } done
 @return { Error | Array } 新刊のBookListコレクション
 **/
-Postman.prototype.inspectSeries = function(seriesItems, done){
-	var i, seriesItem, newReleasies = [];
-	for (i = 0; i < seriesItems.length; i++) {
-		seriesItem = seriesItems[i];
-		newReleasies = this._diffItems(seriesItem.recentContains, seriesItem.currentContains);
-	}
-	if(newReleasies.length === 0){
-		return done({
-			"status": 'not-sent',
-			"message": "通知するシリーズ書籍がありません"
-		});
-	}
-	done(null, newReleasies);
+Postman.prototype.inspectSeries = function inspectSeries(seriesItems, done) {
+  let index;
+  let seriesItem;
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).